Dockerfile

```diff
@@ -1,50 +1,30 @@
 FROM docker.io/golang:alpine as builder
 
-ARG YGG_VERSION=v0.4.7
 ENV CGO_ENABLED=0
 
 WORKDIR /src
 
 RUN apk add git
 
-RUN git clone --depth 1 --branch $YGG_VERSION https://github.com/yggdrasil-network/yggdrasil-go.git .
-RUN ./build && go build -o /src/genkeys cmd/genkeys/main.go
-
 RUN git clone https://github.com/threefoldtech/tfgridclient_proxy && cd tfgridclient_proxy/cmds/proxy_server &&\
     CGO_ENABLED=0 GOOS=linux go build -ldflags "-w -s -X main.GitCommit=$(shell git describe --tags --abbrev=0) -extldflags '-static'"  -o server &&\
     chmod +x server
 
-RUN git clone https://github.com/threefoldtech/rmb_go && cd rmb_go/cmds/msgbusd &&\
-    CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s -extldflags "-static"' -o msgbusd &&\
-    chmod +x msgbusd
-
 FROM alpine:3.14
 
-RUN apk --update add redis 
-
-COPY --from=builder /src/yggdrasil /usr/bin/yggdrasil
-COPY --from=builder /src/yggdrasilctl /usr/bin/yggdrasilctl
-COPY --from=builder /src/genkeys /usr/bin/genkeys
 COPY --from=builder /src/tfgridclient_proxy/cmds/proxy_server/server /usr/bin/server
```

Comment:
let's user a better name than server

Reply:
any suggestions? we didn't yet agree on a new name for the stripped-down gridproxy so I'm not sure here.

Reply:
can be gridproxy or gridrest for now